server/app/controllers/admin/AdminProgramTranslationsController.java

```diff
@@ -128,6 +130,11 @@ private ProgramDefinition getDraftProgramDefinition(String programName) {
    *     same {@link ProgramTranslationView} with error messages
    */
   @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
+  // Adding our custom large body parser. The introduction of the eligibility message
+  // translations has resulted in the occasional exception if the program has a large
+  // number of them. This is a band-aid to allow them to be saved right now, the
```

Comment:
Do we have an issue tracking the redesign? Would it be helpful to include that in this comment?

Reply:
I plan on letting Shreya know and something will be filed